apps/storybook/vite.config.ts

```diff
@@ -1,8 +1,10 @@
 import panda from "@pandacss/dev/postcss";
 import { defineConfig } from "vite";
 import * as path from "path";
+import react from "@vitejs/plugin-react";
 
 export default defineConfig({
+  plugins: process.env._STORYBOOK_DEV ? [] : [react()],
```

Comment:
このコードだけみると環境変数の存在理由が謎なので、以下のコメントを入れてもらいたいです。

```suggestion
  // Enabling react plugin prevents us from running storybook on dev, but required on Chromatic build,
  // So we put our own temporary environment variable (_STORYBOOK_DEV) to tell if it's dev or not to switch plugins.
  plugins: process.env._STORYBOOK_DEV ? [] : [react()],
```

Reply:
ありがとうございます!

Reply:
fixed: https://github.com/tailor-platform/frontend-packages/pull/279/commits/98d6e8b2ee26da3634b706dd42941258c6f0ad90